e2e/commands/add-many.e2e.js

```diff
@@ -118,4 +123,23 @@ describe('bit add many programmatically', function () {
       expect(nodeStartOutputObj).to.be.ofSize(5);
     });
   });
+  describe('make sure gitignore is read from the correct path', function () {
+    before(function () {
+      helper.reInitLocalScope();
+      helper.copyFixtureComponents('add-many');
+      helper.linkNpm('bit-bin');
+      const newDirPath = helper.createNewDirectory();
+      const scriptRelativePath = 'add-many';
+      helper.copyFixtureComponents(scriptRelativePath, newDirPath);
+      helper.linkNpm('bit-bin', newDirPath);
+      helper.writeGitIgnore(['add_many_test_files/c.js']);
```

Comment:
add few more cases for the git ignore
add auto ignored files and make sure they are ignored 
add dist files and check it is ignored
add config files and make sure they are ignored